package chapter14.trycatch;

public class BadIndex3 {
	public static final int NO;
	static {
		NO = 10;
	}

	public static void main(String[] args) {
		int[] array = new int[NO];
		for (int i = 0; i < array.length; i++) {
			array[i] = i;
		}
		int result;
		try {
			System.out.println("1. Program Print Out Part");
		} catch (ArrayIndexOutOfBoundsException e) {
			System.out.println("잘했어요 결과 값은");
		} 
		try {
			System.out.println("1. Program Print Out Part");
		} catch (ArrayIndexOutOfBoundsException e) {
			System.out.println("Print Result");
		} 
	}
}
